crate_universe/src/cli/generate.rs

```diff
@@ -54,7 +55,7 @@ pub struct GenerateOptions {
     #[clap(long)]
     pub repin: bool,
 
-    /// The path to a Cargo metadata `json` file.
+    /// The path to a Cargo metadata `json` file. This file must be next to a `Cargo.toml` and `Cargo.lock` file.
```

Comment:
Even if you're not using a `Cargo.toml` file in your use of `crate_universe` rules, behind the scenes one will always be created and that's what's used to generate this Cargo metadata file. There is still an expectation that the file passed here is effectively within a valid Cargo workspace. Generally users would never have to think about this though. This could probably be improved but it's just where I landed trying to get things working 😄